backend/app/controllers/spree/admin/products_controller.rb

```diff
@@ -116,7 +116,7 @@ def collection
         # @search needs to be defined as this is passed to search_form_for
         # Temporarily remove params[:q][:deleted_at_null] from params[:q] to ransack products.
         # This is to include all products and not just deleted products.
-        @search = @collection.ransack(params[:q].reject { |k, _v| k.to_s == 'deleted_at_null' })
+        @search = @collection.ransack(params[:q].to_unsafe_h.reject { |k, _v| k.to_s == 'deleted_at_null' })
```

Comment:
@mauazua The ransack whitelisting attrs is already part of model doing it again didn't seem right to me. It accepts query param hash and if nothing is whitelisted in that hash it will itself raise exception.